.gitignore

```diff
@@ -52,4 +52,5 @@ bootnode.key
 nodelogs/*
 nodelogs
 
+networkvisualizer/*.dot
```

Comment:
Please don't constrain this to the `networkvisualizer` dir. Any `.dot` file should be ignored in the entire repo